src/qibolab/transpilers/abstract.py

```diff
@@ -1,12 +1,57 @@
 from abc import ABC, abstractmethod
 from typing import List, Optional, Tuple
 
+import networkx as nx
 import numpy as np
 from qibo.backends import NumpyBackend
-from qibo.config import log
+from qibo.config import log, raise_error
 from qibo.models import Circuit
 
 
+# TODO update circuit representation
+def create_circuit_repr(circuit):
+    """Translate qibo circuit into a list of two qubit gates to be used by the transpiler.
```

Comment:
For what I understand, usually the imperative form is preferred in docstrings ([D401](http://www.pydocstyle.org/en/2.1.1/error_codes.html)).
Clearly it's just a question of convention, but since we are talking about standardising the style, we should probably stick to the standard conventions.